components/lib/hooks/useGlobalOnEscapeKey.js

```diff
@@ -5,6 +5,7 @@ import { DomHandler } from '../utils/Utils';
  * Priorities of different components (bigger number handled first)
  */
 export const ESC_KEY_HANDLING_PRIORITIES = {
+    TOOLTIP: 10,
```

Comment:
I wonder if tooltip should be 1200?  I would think tooltips should always be on top of everything else?